### DISCUSSION TITLE
Is there anyway to catch the order details?

### DISCUSSION THREAD

**Discussion by fraudlabspro-com** [medusajs/medusa#10681] [Q&A]
Hi there,

We recently discovered this framework, and was trying to see if we can add our API into this framework to validate the order, however we couldn't get the order details in anyway. Hence we are seeking for help here. Please advise us on how we can get the order details to move on. Many thanks for the help!

**Comment by mrfzd** [comment]
You could use Medusa SDK, https://docs.medusajs.com/resources/js-sdk

**Reply by fraudlabspro-com** [reply]
Hi,

Thank you for the reply. We had explored the order section in the storefront section in the documentation, however we didn't found any methods that come handy to solve our problem. We would like to see if the sdk can helps us to intercept the order details during checkout process, can you please advice us on this?
